publications/admin.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib import admin
+from .models import PDFFile
+
+@admin.register(PDFFile)
+class PDFFileAdmin(admin.ModelAdmin):
+    list_display = ('title', 'author', 'publication_date', 'file_size', 'is_public', 'requires_login', 'uploaded_at', 'updated_at')
+    list_filter = ('is_public', 'requires_login', 'uploaded_at', 'updated_at', 'publication_date')
+    search_fields = ('title', 'slug', 'description', 'author')
+    prepopulated_fields = {'slug': ('title',)}
+    readonly_fields = ('uploaded_at', 'updated_at', 'file_size')
+    fieldsets = (
+        (None, {
+            'fields': ('title', 'slug', 'author', 'publication_date', 'description', 'file')
+        }),
+        ('Access Control', {
+            'fields': ('is_public', 'requires_login'),
```

Comment:
These feel like duplicates

Reply:
One is if anyone (including members) gets to see it and the other is for if you have to be logged in to see it